### PR TITLE
Clean up and organize

### PR DESCRIPTION
**Pull request by javiesses**
- Update index with more links
- Sum up missing stuff
- Created new blank documentation pages to correctly build links

### CODE REVIEW

**Review by jessgusclark — APPROVED** [review]
👍

**Review by ilanolkies — APPROVED** [review]

File: ssi/applications/index.markdown [hunk 1]
```diff
@@ -4,5 +4,6 @@ layout: default
 
 ## Applications
 
-- [Issuer App](./issuer-app) - React application that handles the administrative tasks of issuing and rejecting credential requests - uses [issuer backend](../services) to handle requests and SSI operations
-- [Holder App](./holder-app) - React Native wallet that holds and requests Verifiable Credentials - uses [user-centric data vault]({{ site.baseurl }}/data-vault) service to store backup of user's SSI properties
+- [Issuer app](./issuer-app) - application that serves as the credential request manager. It allows to grant-deny requests or revoke existing credentials
```
Comment (miltonwg) [inline ssi/applications/index.markdown:7]:
```suggestion
- [Issuer app](./issuer-app) - web application that serves as the credential request manager. It allows to grant-deny requests, issue verifiable credentials and revoke existing ones.
```

File: ssi/applications/index.markdown [hunk 2]
```diff
@@ -4,5 +4,6 @@ layout: default
...
+- [Holder app](./holder-app) - wallet used to store declarative details and credentials of it’s users
```
Comment (miltonwg) [inline ssi/applications/index.markdown:8]:
```suggestion
- [Holder app](./holder-app) - mobile wallet used to store verifiable credentials and declarative details of it’s users
```

File: ssi/applications/index.markdown [hunk 3]
```diff
@@ -4,5 +4,6 @@ layout: default
...
+- [Verifier app](./verifier-app) - QR scanner app that verifies Verifiable Presentations
```
Comment (miltonwg) [inline ssi/applications/index.markdown:9]:
```suggestion
- [Verifier app](./verifier-app) - QR scanner mobile app that verifies verifiable presentations (of verifiable credentials)
```

File: ssi/services/index.markdown [hunk 4]
```diff
@@ -6,7 +6,5 @@ layout: default
 
 RIF Self-Sovereign Identity solution provides four main services to interact with:
```
Comment (miltonwg) [inline ssi/services/index.markdown:7]:
```suggestion
RIF Self-Sovereign Identity solution provides two main services to interact with:
```

I see only 2 services listed

File: ssi/services/index.markdown [hunk 5]
```diff
@@ -6,7 +6,5 @@ layout: default
...
 
-- Issuer Back Office - lists all pending credentials and grant/deny each of them
-- Credential Requests - handles all credential requests and emits credentials if they have been granted before
-- Data vault - IPFS pinner/unpinner service and puts/gets/deletes stuff in a centralized key-value DB
-- [Convey](./convey-service) - uploads files to IPFS and caches/serves those cached files
+- [Convey service](./convey-service) - public transport layer for JWTs using IPFS
+- [Issuer service](./issuer-service) - serves for an application that allows receiving credential issuance requests and approving them manually
```
Comment (miltonwg) [inline ssi/services/index.markdown:10]:
```suggestion
- [Issuer service](./issuer-service) - web application that receives verifiable credential issuance requests and lets the user to manually deny or approve them (thus, issuing the verifiable credential)
```

**Review by miltonwg — CHANGES_REQUESTED** [review]
Some changes requested.

File: ssi/services/index.markdown (hunk 5 above)
Comment (ilanolkies) [reply ssi/services/index.markdown:10]: This is not the web app, is serving for the web app.
```suggestion
- [Issuer service](./issuer-service) - serves for a [web application](../apps/issuer-app) that receives verifiable credential issuance requests and lets the user to manually deny or approve them (thus, issuing the verifiable credential)
```

**Review by miltonwg — APPROVED** [review]

**Pull request merged**